src/k8s/pkg/utils/kubectl.go

```diff
@@ -0,0 +1,46 @@
+package utils
```

Comment:
seems like we have done a bit of duplicate work here:
https://github.com/canonical/k8s-snap/pull/57/files

I would not reimplement the go-k8s library but rather - if possible - provide more abstraction.
For example, from my PR: Not implementing "k8s.GetNodes" and then iterate on the node objects but rather "k8s.ClusterReady".